testing/file_test/autoupdate.cpp

```diff
@@ -206,14 +216,18 @@ auto AutoupdateFileTest(
                                     << line_number_replacement.pattern << "`";
 
   // Prepare CHECK lines.
-  llvm::SmallVector<llvm::SmallVector<CheckLine>> stdout_check_lines =
-      BuildCheckLines(stdout, "STDOUT", filenames,
-                      line_number_replacement.has_file, line_number_re,
-                      do_extra_check_replacements);
-  llvm::SmallVector<llvm::SmallVector<CheckLine>> stderr_check_lines =
-      BuildCheckLines(stderr, "STDERR", filenames,
-                      line_number_replacement.has_file, line_number_re,
-                      do_extra_check_replacements);
+  llvm::SmallVector<CheckLine> stdout_check_lines = BuildCheckLines(
+      stdout, "STDOUT", filenames, line_number_replacement.has_file,
+      line_number_re, do_extra_check_replacements);
+  llvm::SmallVector<CheckLine> stderr_check_lines = BuildCheckLines(
+      stderr, "STDERR", filenames, line_number_replacement.has_file,
+      line_number_re, do_extra_check_replacements);
+  auto* stdout_check_line = stdout_check_lines.begin();
+  auto* stderr_check_line = stderr_check_lines.begin();
+
+  bool any_attached_stdout_lines = std::any_of(
```

Comment:
Some tests (lexer tests) will set line_number without file_number. As written, those get associated with file 0, but should probably be treated as unattached. Maybe a TODO? I'm fine fixing it, I think the TODO for handling multi-file remapping is also going to affect multi-file testing so I would probably be picking that up too.